### ISSUE TITLE
BUG: Wrong database when creating backup

### ISSUE THREAD

**Issue by aarbmx6s** [beekeeper-studio/beekeeper-studio#2314]
**Describe the bug**
I have a connection where the default database is `****aidata`

Locally, I have plenty of different databases, some of them:`*****ai` and `*****aidata`.

When I try to create a backup for the `*****aidata` database, after backup configuration, I found that the tool generates a backup for another database — `*****ai`.

I didn't find where I can set the right database name.

**To Reproduce**
1. Connect to some of the databases
2. Start creating a database backup for the current database
3. Choose entities and configure a backup
5. See the execution command where the database argument is wrong

**Expected behavior**
The database should be the current database or somewhere should be an option to choose the needed database.

**Screenshots / Gifs**

The saved connection:
<img width="1202" alt="Screenshot 2024-07-17 at 12 51 55" src="https://github.com/user-attachments/assets/1e841e04-4fe6-4045-93fa-b489fe79b96e">

Backup configuration
<img width="1199" alt="Screenshot 2024-07-17 at 12 56 25" src="https://github.com/user-attachments/assets/4db1a9ac-41bf-416c-a616-be075ed20de0">

<img width="1197" alt="Screenshot 2024-07-17 at 12 59 36" src="https://github.com/user-attachments/assets/88a902e4-1b3b-436d-b0aa-6e2d56b51d2c">

The last step before execution:
<img width="1195" alt="Screenshot 2024-07-17 at 13 00 49" src="https://github.com/user-attachments/assets/99a57cdd-13ff-46d8-91c7-b12d32a92e3a">

**Version Information (please complete the following information):**
 - OS: MacOS
 - App Version: Mac
 - Database type and version: Posgresql 13

**Comment by not-night-but** [comment]
Yeah this is definitely a bug. Weird that it's pulling tables from the right database but not the right one for the create flag. I'm away for today, but I'll look into this when I get a chance tomorrow, should be an easy fix. Thanks for the report!
